package.json

```
@@ -35,7 +35,7 @@
     "url": "https://github.com/DavidWells/markdown-magic"
   },
   "dependencies": {
-    "@technote-space/doctoc": "^2.4.7",
+    "@technote-space/doctoc": "2.4.7",
```

Since this project doesn't pin dependencies anywhere (it would be great if it did), it might be more natural to use some range instead:
```suggestion
    "@technote-space/doctoc": "~2.4",
```